app/controllers/forem/api/v1/forums_controller.rb

```diff
@@ -0,0 +1,13 @@
+require_dependency "forem/application_controller"
+
+module Forem
+  module Api
+    module V1
+      class ForumsController < ApplicationController
+        load_and_authorize_resource :class => 'Forem::Forum', :only => :show
+
+        before_action { request.format = :json }
```

Comment:
`respond_to` doesn't seem to address the problem I was trying to solve. The problem is that I'm using a custom MIME type (`application/vnd.forem+json`), as recommended in any number of blog posts, but because of that, Rails looks for a view named `show.application/vnd.forem+json.jbuilder`. I just wanted to call it `show.json.jbuilder`.

I tried `Mime::Type.register 'application/vnd.forem+json', :json`, which works, but I get a warning about initializing `Mime::JSON` twice. There doesn't seem to be a way to say "Just treat this MIME type as JSON, too." (`register_alias` doesn't do it.)

It looks like a solution is to register the MIME type as `:forem` and then rename the views to `show.forem.jbuilder`. But it seems like there should be a way to use `.json`.